homeassistant/components/shopping_list/__init__.py

```diff
@@ -88,6 +97,18 @@ async def add_item_service(call: ServiceCall) -> None:
         if (name := call.data.get(ATTR_NAME)) is not None:
             await data.async_add(name)
 
+    async def remove_item_service(call: ServiceCall) -> None:
+        """Remove an item with `name`."""
+        data = hass.data[DOMAIN]
+        if (name := call.data.get(ATTR_NAME)) is None:
```

Comment:
What's with the None-check, why isn't a non-None `name` required by the service's schema? Same comment for the existing services.

Reply:
I do not see a reason for that. The services that do not require a `name` as input have a separate empty schema anyway. I will remove the `None`.